win32ss/gdi/ntgdi/freetype.c

```diff
@@ -5490,8 +5491,9 @@ NtGdiGetFontFamilyInfo(HDC Dc,
     }
 
     /* Allocate space for a safe copy */
-    DataSize = SafeInfoCount * sizeof(FONTFAMILYINFO);
-    if (DataSize <= 0)
+    DataSize64 = UInt32x32To64(SafeInfoCount, sizeof(FONTFAMILYINFO));
```

Comment:
I'd really prefer intsafe functions, as they make for shorter and less error-prone code.

Reply:
Identifier `RtlLongMult` is not defined in GCC.

Reply:
OK. I will use `RtlULongMult` instead.